gtk/src/light/gtk-3.20/_tweaks.scss

```diff
@@ -62,254 +62,112 @@ $small_radius: 4px;
   }
 }
 
-headerbar,
-.titlebar {
-  @include yaru_headerbar_fill($headerbar_bg_color, $hc: $headerbar_border_color);
-  color: $headerbar_fg_color;
-
-  label .title {
-    color: $headerbar_fg_color;
-  }
-
-  &:backdrop {
-    border-color: mix(darken($inkstone, 11%), darken($inkstone, 1%), 80%);
-    background-color: $backdrop_headerbar_bg_color;
-    box-shadow: inset 0 1px $headerbar_border_color;
-  }
-
-  .maximized &,
-  .fullscreen & {
-    box-shadow: none;
-  }
-
-  button,
-  button.image-button,
-  button.image-button.toggle,
-  button.text.button,
-  button.toggle,
-  box button.toggle,
-  buttonbox.toggle,
-  stackswitcher button,
-  filechooser .path-bar.linked>button,
-  .path-bar button {
-    $_btn_backdrop_border: darken($headerbar_border_color, 3%);
-
-    @include button(normal, lighten($headerbar_bg_color, 4%), $headerbar_fg_color);
-
-    &.flat { @include button(undecorated); }
-
-    &:hover {
-      @include button(hover, lighten($headerbar_bg_color, 8%), $headerbar_fg_color);
-    }
-
-    &:active,
-    &:checked,
-    &.toggle:checked,
-    &.toggle:active {
-      $_bg: lighten($headerbar_bg_color, if($variant==light, 0%, 1%));
-      @include button(active, $_bg, $headerbar_fg_color);
-      border-color: rgb(20, 19, 19);
-      background-image: image(darken($_bg, 9%));
-    }
-
-    &:backdrop {
-
-      &.flat,
-      & {
-        @include button(backdrop, $backdrop_headerbar_bg_color, $backdrop_headerbar_fg_color);
-
-        -gtk-icon-effect: none;
-        border-color: $_btn_backdrop_border;
-
-        &:active,
-        &:checked {
-          $_bg: if($variant=='light', darken($backdrop_headerbar_bg_color, 9%), lighten($backdrop_headerbar_bg_color, 2%));
-          @include button(backdrop-active, $_bg, $backdrop_headerbar_fg_color);
-          border-color: $_btn_backdrop_border;
-        }
-
-        &:disabled {
-          @include button(backdrop-insensitive, if($variant=='light', darken($headerbar_bg_color, 15%), $headerbar_bg_color), $backdrop_headerbar_fg_color);
-
-          border-color: $_btn_backdrop_border;
-
-          &:active,
-          &:checked {
-            @include button(backdrop-insensitive-active, $backdrop_headerbar_bg_color, $backdrop_headerbar_fg_color);
-
-            border-color: $_btn_backdrop_border;
-          }
-        }
+headerbar.titlebar {
```

Comment:
I did not test it yet, but are you sure about this change? You are applying now the style only to `headerbar.titlebar` while in previous styling it was to both `headerbar` and `.titlebar` separately, so even `headerbar` withouth `.titlebar` had this style.

Reply:
I applied the background again to only titlebar with extending %yaru_titlebar, but yes, it's weird. Any improvement is welcome :D